src/js/components/stock-movement-wizard/request/EditPage.jsx

```diff
@@ -1033,6 +1033,7 @@ class EditItemsPage extends Component {
         quantityOnHand: val.quantityOnHand > 0 ? val.quantityOnHand : 0,
         quantityAvailable:
             val.quantityAvailable > 0 ? val.quantityAvailable : 0,
+        quantityOnHandRequesting: val.quantityCounted,
```

Comment:
if you keep using `quantityCounted` everywhere, why here it is named as `quantityOnHandRequesting`?

Reply:
additional question - where is it even used? I can see you adding it here, but I don't see it being used anywhere (backend would anyway skip it, because you name it as `quantityCounted` everywhere)

Reply:
I just followed the naming of the column. It was `quantityOnHandRequesting` in all of the fields in this component.

Reply:
after looking into the code, I get it now - I haven't known it's a read-only field - this is why I was confused on the naming.